bfh/transformations.py

```diff
@@ -365,9 +365,14 @@ class DateToIsoString(Transformation):
     Turn a datetime into an ISO 8601 formatted string.
 
     """
+    null_types = (None,)
+
     def function(self, source, *call_args):  # source ignored
         try:
-            return call_args[0].isoformat()
+            value = call_args[0]
+            if value in self.null_types:
+                return value
+            return value.isoformat()
```

Comment:
no, because we are also trying to catch the `IndexError`, in case no argument is passed at all.